augur/tree.py

```diff
@@ -232,8 +232,22 @@ def random_string(n):
             tmp_line = line
             if line.startswith(">"):
                 num_seqs += 1
-                for c,v in escape_dict.items():
-                    tmp_line = tmp_line.replace(c,v)
+                pos = 1
+                for i in range(1, len(line)):
+                    l = line[i]
+                    #check if l is a character that IQ-tree changes
+                    if l.isalnum()==False and l !='_' and l != '-' and l != '.' and l.isspace()== False:
```

Comment:
You used outdated IQtree code unfortunately, I think there have been recent changes, adding `\` or `/`:
https://github.com/iqtree/iqtree2/blob/3bbc304263cb2f85574a9163e8f2e5c5b597a147/utils/tools.cpp#L585

Maybe link to the IQtree source code in a comment so that in the future, if IQtree changes escaped characters it's easy to find.

You looked up the function in an old repo of IQtree

Reply:
Actually, the change seems to be that `|` is accepted, but since this seems to be already handled by treetime we don't have an issue here.

But still good to link to the current IQtree source code I think.
```suggestion
                    # See IQtree function renameBool https://github.com/iqtree/iqtree2/blob/3bbc304263cb2f85574a9163e8f2e5c5b597a147/utils/tools.cpp#L585
                    if l.isalnum()==False and l !='_' and l != '-' and l != '.' and l.isspace()== False:
```

Reply:
Oh great you caught this!